_opensearch/rest-api/nodes-apis/nodes-info.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: default
+title: Nodes info
+parent: Nodes APIs
+grand_parent: REST API reference
+nav_order: 20
+---
+
+# Nodes info
+
+Represents mostly static information about your cluster's nodes, including but not limited to:
+
+- Host system information 
+- JVM 
+- Processor Type 
+- Node settings 
+- Thread pools settings 
+- Installed plugins
+
+## Example
+
+To get information on all nodes in a cluster:
+
+```bash
+GET /_nodes
+```
+
+To get thread pool information from the cluster manager node only:
+
+```bash
+GET /_nodes/master:true/thread_pool?pretty
+```
+
+## Path and HTTP methods
+
+```bash
+GET /_nodes
+GET /_nodes/{nodeId}
+GET /_nodes/{metrics}
+GET /_nodes/{nodeId}/{metrics}
+# or full path equivalent
+GET /_nodes/{nodeId}/info/{metrics}
+```
+
+## URL parameters
+
+You can include the following URL parameters in your request. All parameters are optional.
+
+Parameter | Type   | Description
+:--- |:-------| :---
+nodeId | String | A comma-separated list of nodeIds to filter results. Supports [node filters](../index/#node-filters). Defaults to `_all`.
+metrics | String | A comma-separated list of metric groups that will be included in the response. For example `jvm,thread_pools`. Defaults to all metrics.
+timeout | TimeValue | A request [timeout](../index/#timeout). Defaults to `30s`.
+
+The following are listed for all available metric groups:
+
+Metric | Description
+:--- |:----
+`settings` | A node settings. This is combination of the default settings, custom settings from [configuration file](../../../configuration/#configuration-file) and dynamically [updated settings](../../../configuration/#update-cluster-settings-using-the-api).
```

Comment:
Specifies the settings for a node.
(also make full sentences out of all points, or remove period)